deploy/templates/bicep/Create-IomtContainerAppEnv.ps1

```diff
@@ -0,0 +1,114 @@
+param
+(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [ValidateLength(3,16)]
+    [ValidateScript({
+        if ("$_" -cmatch "(^([a-z]|\d)+$)") {
+            return $true
+        }
+        else {
+			throw "Service name must be lowercase and numbers"
+            return $false
+        }
+    })]
+    [string]$baseName,
+    [Parameter(Mandatory = $true)]
+    [ValidateSet(
+        'australiaeast', 
+        'canadacentral', 
+        'eastus', 
+        'eastus2', 
+        'japaneast', 
+        'koreacentral', 
+        'northcentralus',
+        'northeurope',
+        'qatarcentral',
+        'southcentralus',
+        'southeastasia',
+        'swedencentral',
+        'switzerlandnorth',
+        'westcentralus',
+        'westeurope',
+        'westus2',
+        'westus3',
+        'uksouth'
+   )]
+    [string]$location,
+    [Parameter(Mandatory = $true)]
+    [ValidateSet(
+        'Create',
+        'Lookup',
+        'LookupWithEncounter'
+    )]
+    [string]$resourceIdentityResolutionType
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+# Get current Az context
+try {
+    Write-Host "Get current Az context..."
+    az account show 
+} 
+catch {
+    throw "Please log in with az login cmdlet before proceeding"
+}
+
+# Get current account context - User/Service Principal
+$azAccountId = az account show --query user.name --output tsv
+$azAccountType = az account show --query user.type --output tsv
+if ($azAccountType -eq "user") {
+    Write-Host "Current account context is user: $($azAccountId)"
+    
+    $currentUser = az ad user show --id $azAccountId #Get-AzADUser -UserPrincipalName $azContext.Account.Id 
+
+    if ($currentUser) {
+        $currentObjectId = az ad user show --id $azAccountId --query id --output tsv
```

Comment:
is this variable being used anywhere?

Reply:
$currentObjectId is being used to validate that the user has an objectId, otherwise an error will be thrown. However, I removed another line where the value of $currentObjectId is set, but never used. Thanks for pointing this out.